Makefile

```diff
@@ -0,0 +1,3 @@
+copy:
+	cp ./src/spritesh ./bin/npm/spritesh
+	cp ./src/spritesh ./bin/gem/sprite.sh
```

Comment:
Because that's the name of the executable. See [this](http://stackoverflow.com/a/27988355/1679327) for more info.